packages/db-postgres/src/schema/withDefault.ts

```diff
@@ -0,0 +1,17 @@
+import type { PgColumnBuilder } from 'drizzle-orm/pg-core'
+import type { FieldAffectingData } from 'payload'
+
+export const withDefault = (
+  column: PgColumnBuilder,
+  field: FieldAffectingData,
+): PgColumnBuilder => {
+  if (typeof field.defaultValue === 'undefined' || typeof field.defaultValue === 'function')
+    return column
+
+  if (typeof field.defaultValue === 'string' && field.defaultValue.includes("'")) {
```

Comment:
how come we are checking for `field.defaultValue === 'string'` here? Shouldn't numbers, etc also be valid?

Reply:
This is just if we need to escape `'` in strings